chain/types/message.go

```diff
@@ -21,7 +21,7 @@ type Message struct {
 	Value BigInt
 
 	GasPrice BigInt
-	GasLimit BigInt
+	GasLimit int64
```

Comment:
Is there some discussion on why this is int64 vs uint? Is there any use in having negative values here?

Reply:
Yeah, I think the spec team expects there to be cases where we have something that costs negative gas, and making all the types support signed values makes that easier